### DISCUSSION TITLE
telnetlib connection

### DISCUSSION THREAD

**Discussion by iAmJamesGardner** [pyscript/pyscript#1182] [Packages]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

I tested importing and making a connection with telnetlib. The script works outside of the HTML page. Inside, the browser displays the following error:

JsException(PythonError: Traceback (most recent call last): File "/lib/python3.10/site-packages/_pyodide/_base.py", line 429, in eval_code .run(globals, locals) File "/lib/python3.10/site-packages/_pyodide/_base.py", line 300, in run coroutine = eval(self.code, globals, locals) File "", line 7, in File "/lib/python3.10/telnetlib.py", line 235, in open self.sock = socket.create_connection((host, port), timeout) File "/lib/python3.10/socket.py", line 845, in create_connection raise err File "/lib/python3.10/socket.py", line 833, in create_connection sock.connect(sa) BlockingIOError: [Errno 26] Operation in progress )

### What browsers are you seeing the problem on? (if applicable)

Chrome, Microsoft Edge

**Comment by marimeireles** [comment]
Pyodide doesn't offer support for the networking stack yet, which makes me believe that's why it's not working for you.
Here is some related WIP in the Pyodide repo: https://github.com/pyodide/pyodide/issues/398
We're planning to work on this =) but no plans for exactly when
